apps/daimo-mobile/src/view/shared/SearchHeader.tsx

```diff
@@ -39,25 +37,27 @@ export function SearchHeader({
     isFocused.value = prefix != null;
   }, [prefix]);
 
-  const qrButton = useAnimatedStyle(() => {
+  const accountButton = useAnimatedStyle(() => {
```

Comment:
looks like this should still be called `qrButton` based on how it's used below

Reply:
True. Updated in latest commit.